ampd/src/starknet/json_rpc.rs

```diff
@@ -68,9 +71,9 @@ pub trait StarknetClient {
 
     /// Attempts to fetch a SignersRotated event, by a given `tx_hash`.
     /// Returns a tuple `(tx_hash, event)` or a `StarknetClientError`.
-    async fn get_event_by_hash_signers_rotated(
+    async fn get_event_by_hash_and_index_signers_rotated(
```

Comment:
Can you please rename the funcs to `get_event_by_message_id_signers_rotated` and `get_event_by_message_id_contract_call` if it's not too much trouble?